internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,59 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parse := strings.Split(data, ",")
+
+	if len(parse) != 2 {
+		return 0, 0, errors.New("длина слайса меньше 2")
+	}
+	// конвертируем шаги в число, если ошибка то вызываем ошибку
+	steps, err := strconv.Atoi(parse[0])
+	if err != nil {
+		return 0, 0, errors.New("ошибка преобразования в целое число")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```